app/TextParser/TableCurrencySummary.php

```diff
@@ -51,7 +51,7 @@ public function process()
 			}
 			if (!empty($currency) && !empty($currencyData) && $baseCurrency['id'] !== $currency && $inventory->isField('tax') && $inventory->isField('taxmode') && $inventory->isField('currency')) {
 				$RATE = $baseCurrency['conversion_rate'] / $currencyData['conversion_rate'];
-				$html .= '<table class="productTable colapseBorder">
+				$html .= '<table style="border-collapse:collapse;width:100%;border:1px solid #ddd;">
```

Comment:
don't use "style"

Reply:
I must, because YetiForcePDF doesn't support classes right now, it's the only way